addons/sale/report/sale_report_views.xml

```diff
@@ -88,7 +88,17 @@
         <field name="view_mode">graph,pivot</field>
         <field name="view_id"></field>  <!-- force empty -->
         <field name="search_view_id" ref="view_order_product_search"/>
-        <field name="context">{'search_default_Sales':1, 'group_by_no_leaf':1,'group_by':[]}</field>
+        <field name="context">{'search_default_Sales':1, 'group_by_no_leaf':1, 'group_by':[]}</field>
+        <field name="help">This report performs analysis on your quotations and sales orders. Analysis check your sales revenues and sort it by different group criteria (salesman, partner, product, etc.) Use this report to perform analysis on sales not having invoiced yet. If you want to analyse your turnover, you should use the Invoice Analysis report in the Accounting application.</field>
+    </record>
```

Comment:
Remove the diff on those lines since you are not modifying this action anymore, only adding a new one.
It might ease the fw-ports, and it'll be clearer to see the true changes of your commit :).